### PR TITLE
daypicker compenent update for color token

### PR DESCRIPTION
**Pull request by yanliwa**
# Links
https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=44859%3A103117

UI
Original             	   |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **
*image|image

### CODE REVIEW

File: platformcomponents/desktop/date-picker.json [hunk 1]
```diff
@@ -39,22 +39,45 @@
...
+          },
+          "#pressed": {
+            "background": "@theme-button-secondary-pressed",
+            "text": "@theme-text-primary-normal",
+            "border": "@theme-outline-button-normal"
+          },
+          "#disabled": {
+            "background": "@theme-button-secondary-normal",
+            "text": "@theme-text-primary-disabled",
+            "border": "@theme-outline-button-normal"
```
Comment (nataliadelmar) [inline platformcomponents/desktop/date-picker.json:61]: I see outline-disabled-normal in the figma for the border in disabled mode

File: platformcomponents/desktop/date-picker.json [hunk 2]
```diff
@@ -39,22 +39,45 @@
...
-          "#hovered": "@theme-text-primary-normal",
-          "#pressed": "@theme-text-primary-normal",
-          "#disabled": "@theme-text-primary-disabled",
-          "#active": "@theme-common-text-white"
-        },
-        "border": {
-          "#normal": "@theme-outline-button-normal"
+        "checked": {
+          "#normal": {
+            "background": "@theme-background-accent-normal",
```
Comment (nataliadelmar) [inline platformcomponents/desktop/date-picker.json:66]: I see theme-button-accent-normal in the figma.. theme-background-accent-normal doesn't exist in theme tokens.

File: platformcomponents/desktop/date-picker.json [hunk 3]
```diff
@@ -39,22 +39,45 @@
...
+            "text": "@theme-common-text-white"
+          },
+          "#hovered": {
+            "background": "@theme-background-accent-hover",
```
Comment (nataliadelmar) [inline platformcomponents/desktop/date-picker.json:70]: I see theme-button-accent-hover in the figma.. theme-background-accent-hover doesn't exist in theme tokens.

File: platformcomponents/desktop/date-picker.json [hunk 4]
```diff
@@ -39,22 +39,45 @@
...
+            "text": "@theme-common-text-white"
+          },
+          "#pressed": {
+            "background": "@theme-background-accent-pressed",
```
Comment (nataliadelmar) [inline platformcomponents/desktop/date-picker.json:74]: I see theme-button-accent-pressed in the figma.. theme-background-accent-pressed doesn't exist in theme tokens.

**Review by nataliadelmar — CHANGES_REQUESTED** [review]

File: platformcomponents/desktop/date-picker.json [hunk 5]
```diff
@@ -39,22 +39,45 @@
...
-          "#hovered": "@theme-text-primary-normal",
-          "#pressed": "@theme-text-primary-normal",
-          "#disabled": "@theme-text-primary-disabled",
-          "#active": "@theme-common-text-white"
-        },
-        "border": {
-          "#normal": "@theme-outline-button-normal"
+        "checked": {
+          "#normal": {
+            "background": "@theme--background-accent-normal-normal",
```
Comment (nataliadelmar) [inline platformcomponents/desktop/date-picker.json:66]: it is theme-button-accent-normal, not theme-background-accent-normal-normal... same for the others

**Review by nataliadelmar — CHANGES_REQUESTED** [review]

File: platformcomponents/desktop/date-picker.json [hunk 6]
```diff
@@ -39,22 +39,45 @@
     },
     "dayPicker":{
       "day":{
-        "background":{
-          "#normal": "@theme-button-secondary-normal",
-          "#hovered": "@theme-button-secondary-hover",
-          "#pressed": "@theme-button-secondary-pressed",
-          "#disabled": "@theme-button-secondary-normal",
-          "#active": "@theme-background-accent-normal"
+        "unchecked": {
```
Comment (nataliadelmar) [inline platformcomponents/desktop/date-picker.json:42]: it is not a checkbox, se we can call this 'inactive' and the other one 'active'

**Review by nataliadelmar — CHANGES_REQUESTED** [review]

File: platformcomponents/desktop/date-picker.json (hunk 6 above)
Comment (yanliwa) [reply platformcomponents/desktop/date-picker.json:42]: ok

**Review by nataliadelmar — APPROVED** [review]

**Pull request merged**